### PR TITLE
created todoLoad action with start, success and error handler. Remove…

### PR DESCRIPTION
**Pull request by Theu**
… selectors, axios calls and types that are not needed at the moment

### CODE REVIEW

File: src/redux/actions/actionTypes.js [hunk 1]
```diff
@@ -1,3 +1,3 @@
-export const LOAD_TODOS = 'LOAD_TODOS';
-export const LOAD_TODOS_SUCCESS = 'LOAD_TODOS_SUCCESS';
-export const LOAD_TODOS_ERROR = 'LOAD_TODOS_ERROR';
+export const LOAD_TODOS_REQUEST = 'LOAD_TODOS_START';
```
Comment (BonDoQ) [inline src/redux/actions/actionTypes.js]: wrong name mapping  why not consistent ?

File: src/App.js [hunk 2]
```diff
@@ -2,50 +2,66 @@ import React from 'react';
...
 function mapStateToProps(state) {
-  console.log('state', state);
   return {
-    todos: getTodos(state)
-  };
-}
+    isLoading: isLoading(state),
+    todoList: getTodos(state),
+
+    isError: hasError(state),
```
Comment (BonDoQ) [inline src/App.js]: what "isError" means ? and why not hasError consistent ?

File: src/redux/reducers/todoReducer.js [hunk 3]
```diff
@@ -1,24 +1,36 @@
...
+            return {
+                ...state,
+                isLoading: false,
                 todos: action.payload
             }
+        
+        case LOAD_TODOS_FAILURE:
+            return {
+                ...state,
+                error: [action.payload.message, action.payload.response]
```
Comment (BonDoQ) [inline src/redux/reducers/todoReducer.js]: error is single and it holds array, why ?

File: src/redux/actions/actionTypes.js (hunk 1 above)
Comment (Theu) [reply src/redux/actions/actionTypes.js]: got trapped on IDE

**Pull request merged**